### ISSUE TITLE
DB equivalence with sklearn.metrics.davies_bouldin_score

### ISSUE THREAD

**Issue by AP6YC** [AP6YC/ClusterValidityIndices.jl#58]
The implementation of the Davies-Bouldin (DB) metric is not equivalent to the implementation with the one on scikit-learn.
